src/dtoannotations/Main.java

```diff
@@ -0,0 +1,16 @@
+package dtoannotations;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IllegalAccessException {
+        Validator validator = new Validator();
+        LocalDate localDate = LocalDate.of(2019, 7, 9);
```

Comment:
Please set meaningful variable names. Here can be **birthdayDate**

Reply:
The same for **FOR LOOP**. Change variable name  [ **l** ].

Reply:
done